Projects/Server/Geometry/Point2D.cs

```diff
@@ -112,4 +110,20 @@ public int CompareTo(IPoint2D other)
         var xComparison = m_X.CompareTo(other.X);
         return xComparison != 0 ? xComparison : m_Y.CompareTo(other.Y);
     }
+
+    public bool TryFormat(Span<char> destination, out int charsWritten, ReadOnlySpan<char> format, IFormatProvider provider)
+    {
+        return destination.TryWrite(provider, $"({m_X}, {m_Y})", out charsWritten);
+    }
+
+    public override string ToString() {
+        return $"{this}";
```

Comment:
Is this better than what we had before with: `public override string ToString() => $"({m_X}, {m_Y})";`
I think it will end up being the same, or worse, incur an intermediate buffer before creating the final string.